index.html

```diff
@@ -20,160 +20,165 @@
   </head>
 
   <body>
-    <div class="header">
+    <section class="header">
+      <a href="#">
+        <img
+          src="images/bikes-for-refugees_logo.jpg"
+          alt="Logo for Bikes for Refugees Scotland"
+          class="header__logo"
+        />
+      </a>
+
+      <section>
+        <ul class="navigation__list">
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">Home</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">About Us</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">News & Events</a>
+          </li>
+          <li class="navigation__item">
+            <a class="navigation__link" href="#">Contact Us</a>
+          </li>
+        </ul>
+      </section>
+
+      <button class="dovo">Donate Now</button>
+    </section>
+
+    <section class="content">
+      <section class="main">
```

Comment:
As a rule of thumb, try to keep `<section>` elements only for blocking out sections of your web page. The [MDN Web Docs page](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) suggest only using a section where you can use a heading element (`<h1>`, `<h2>`, etc.), with a few exceptions for grouping together content.

For example, on this page you could use a `<section>` for "Learn more" and "Upcoming events". And you might also use a `<section>` to separate out the 'hero' area of the page - even if that doesn't have a dedicated heading.

Elsewhere, you can use `<div>` sections to add styles to the page